services/api/src/typeDefs.js

```diff
@@ -562,6 +577,10 @@ const typeDefs = gql`
     Returns the costs for all billing groups
     """
     allBillingGroupsCost(month: String!): JSON
+    """
+    Returns the Billing Group Modifiers for a given Billing Group (all if no month filter defined)
```

Comment:
`all if no month filter defined` < should that be "all if no GroupInput filter is defined"?

Reply:
Hmm, my thought was that a user would want all billing modifiers returned for a Billing Group if they omit a month input variable. I can't think of a use case, outside of internal audits, where we'd want to get/display all billing modifiers for all groups. 

Now that I'm thinking about it, that's why I originally named this as `billingModifiers` and not `allBillingModifiers` because we were returning the billingModifiers for a specific Billing Group and not all Modifiers for all groups. 

It might make sense to provide queries for both, I'll need to think on that more.